web/ui/react-app/src/Navbar.tsx

```diff
@@ -25,6 +34,11 @@ const Navigation: FC<PathPrefixProps> = ({ pathPrefix }) => {
       </Link>
       <Collapse isOpen={isOpen} navbar style={{ justifyContent: 'space-between' }}>
         <Nav className="ml-0" navbar>
+          {consoleLink !== '' && (
```

Comment:
Can we just do ```CONSOLES_LINK === 'CONSOLES_LINK_PLACEHOLDER' || CONSOLES_LINK === '' ? null : <NavItem>...</NavItem>``` directly instead of introducing an extra variable?
What if the result of https://github.com/prometheus/prometheus/pull/6761/files#diff-73cbd2009b0e4f156e8ca0f47e95b016R385 is null? Maybe we should check against that as well

Reply:
Combined with my other comment about not having the global variables in this file, I think the extra `consolesLink` variable would make sense, but as a prop, and as type `string | null` (or `string | undefined`, we should at some point standardize on one).

Reply:
Yeah, that's why I suggested this https://github.com/prometheus/prometheus/pull/6761#discussion_r375800896
IMO the confusion around those 'magic' variables will disappear this way.

Reply:
Yeah, fine to add a `GLOBAL_` prefix, but I still think any global state shouldn't even be used outside of index.html.